.env

```
@@ -7,4 +7,9 @@ BETTER_AUTH_URL=http://localhost:3000 # Base URL of your app
 #google generative ai
 GOOGLE_GENERATIVE_AI_API_KEY=AIzaSyDm3BlLdzb1LVGnjr9SQgVXnJ1LWXyiBQ0
 
-SENTRY_AUTH_TOKEN=cdMCqeHcuOv8+kkYphmsNJqUg
\ No newline at end of file
+SENTRY_AUTH_TOKEN=cdMCqeHcuOv8+kkYphmsNJqUg
+
+# Polar
+POLAR_ACCESS_TOKEN="polar_oat_z5HEajR35MBio87fxHE25viFUyteHsSCrKNZ90BciwG"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)